### ISSUE TITLE
Lint/PercentStringArray: Remove autocorrect

### ISSUE THREAD

**Issue by jzinn** [rubocop-hq/rubocop#5660]
The double quotes in the following are intentional:

```ruby
x = %w[
  "C:\\Program\ Files\\foo.exe"
  --option
]
```

When autocorrecting, Rubocop should not remove the quotes, because that changes the meaning of the program.

--------

## Expected behavior

Quotes should remain.

## Actual behavior

Quotes are removed:

```ruby
x = %w[
  C:\\Program\ Files\\foo.exe
  --option
]
```

## Steps to reproduce the problem

```shell
echo '%w["a" "b" "c"]' > foo.rb
cat foo.rb
rubocop -a foo.rb
cat foo.rb
```

## RuboCop version

```
$ rubocop -V
0.53.0 (using Parser 2.5.0.3, running on ruby 2.4.3 x86_64-linux)
```

**Comment by mikegee** [comment]
I confirmed this.

The message is: "Lint/PercentStringArray: Within %w/%W, quotes and ',' are unnecessary and may be unwanted in the resulting strings."

I suppose the cop's author thought they were more likely to be unintended when deciding to remove them in autocorrect, even though Rubocop can't know that.

**Comment by pocke** [comment]
I think the auto-correction has been disabled by default. Because other unsafe auto-corrections are disabled.
